Testscripts/Linux/utils.sh

```diff
@@ -3409,6 +3411,16 @@ function CheckInstallLockUbuntu() {
     fi
 }
 
+function CheckInstallLockSLES() {
+    if pidof zypper;then
+        LogMsg "Another install is in progress. Waiting 1 seconds."
+        sleep 1
+        CheckInstallLockSLES
```

Comment:
This recursive call might fall into an infinite loop. Please add one argument for the counter. Sometimes the lock is there forever.

Reply:
If this happens, this will be a bug on the VM.

Reply:
Your intention is then leaving the timeout logic, and let LISAv2 infra handle the exception. right? I think that is not bad, it only takes longer run-time.